hack/generate/olm_bindata.sh

```diff
@@ -1,9 +1,18 @@
 #!/usr/bin/env bash
 
+function version_gt() {
+    test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1";
+}
+
 function get_olm_manifests() {
     echo "downloading olm manifests for version ${1}"
-    curl -L -o olm-manifests/$1-olm.yaml "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${1}/olm.yaml"
-    curl -L -o olm-manifests/$1-crds.yaml "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${1}/crds.yaml"
+    tag=$1
+    if version_gt ${1} $base_version; then
```

Comment:
The `$base_version` is not defined here. It should be 0.16.1.
`version_gt()` checks if the provided version is greater than base version, and if so appends `v`.  Since olm has started appending `v` for versions after 0.16.1, the base_version should be that value.

Reply:
fixed.